content/cockpit-143.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 143
+author: Martin Pitt
+date: 2017-06-19 14:00
+tags: cockpit, linux
+slug: cockpit-143
+category: release
+summary: Cockpit with new Software Updates page
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 143.
+
+### "Software Updates" page for package based operating systems
+
+There is a new "Software Updates" page for classic dnf/yum/apt based systems,
+to complement the already existing one for OSTree on
+[Atomic](https://www.projectatomic.io). This uses the [PackageKit
+API](https://www.freedesktop.org/software/PackageKit/) for platform
+independence. This feature is shipped in the new `cockpit-packagekit`
+sub-package.
+
+See it in action:
+
+<iframe width="720" height="540" src="https://www.youtube.com/embed/BENWI-dTKMs?rel=0" frameborder="0" allowfullscreen></iframe>
+
+### Inline VNC consoles
+
+The inline VNC console advertised in [Cockpit 142](http://cockpit-project.org/blog/cockpit-142.html)
+was not actually working from the released `cockpit-machine` package due to a
+packaging problem. Cockpit 143 corrects this problem. Thanks to Marek Libra!
+
+### Kubernetes RBAC authorization support
+
+The Kubernetes page can now be used to log into Kubernetes installations that
+use [RBAC authorization](https://kubernetes.io/docs/admin/authorization/rbac/).
+
+### RPM packages for IBM Z systems
+
+The `cockpit-docker` RPM package is now being built on the s390x (IBM Z
+Systems) architecture, as there is a working Fedora Docker package now.
```

Comment:
Good point. How about ... ", as the Fedora Docker package has been verified to work
there." instead?